src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -275,4 +280,22 @@ AbstractTableGenerator::SortOrderByTable AbstractTableGenerator::_sort_order_by_
 void AbstractTableGenerator::_add_constraints(
     std::unordered_map<std::string, BenchmarkTableInfo>& table_info_by_name) const {}
 
+bool AbstractTableGenerator::_table_is_sorted_by(const std::shared_ptr<Table>& table,
+                                                 const SortColumnDefinition& sort_definition) {
```

Comment:
The "sorted" property of tables and chunks is somewhat incompatible. If a table is sorted on multiple columns using the `Sort` operator, it is first sorted according to one column, then to the other. For a chunk, it means this:

```
   * In a chunk, multiple segments may be sorted independently. For example, in a table storing orders, both the order
   * id and date of incoming orders might have incrementing values. In this case, sorted_by has two entries (assuming
   * this knowledge is available to the database). However, for cases where the data is first orderered by one column,
   * then by another (e.g. ORDER_BY last_name, first_name), only the primary order is stored.
```

I can see how this is misleading. Maybe we should rename the chunk method to `individually_sorted_by`?

Reply:
Seems like good idea